pkgs/development/python-modules/nimbo/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, python3Packages, fetchFromGitHub, awscli }:
+
+with python3Packages;
+
+buildPythonApplication rec {
```

Comment:
This is a often used pattern for Python application (at least recently but there are others):

```bash
{ lib
, fetchFromGitHub
, python3
}:

python3.pkgs.buildPythonApplication rec {
  pname = "xyz";
  version = "5.0.0";
  disabled = python3.pythonOlder "3.6";
[...]
  propagatedBuildInputs = with python3.pkgs; [
    beautifulsoup4
  ];
```